on-chain/evm-contracts/contracts/Bridge.sol

```diff
@@ -318,31 +316,26 @@ contract Bridge {
 
     function createDepositProposal(uint destinationChainID, uint depositNonce, bytes32 dataHash) public _onlyRelayers {
         require(_depositProposals[destinationChainID][depositNonce]._status == DepositProposalStatus.Inactive ||
-        _depositProposals[destinationChainID][depositNonce]._status == DepositProposalStatus.Denied, "this proposal is either currently active or has already been passed/transferred");
+            _depositProposals[destinationChainID][depositNonce]._status == DepositProposalStatus.Denied,
+            "this proposal is currently active or has already been passed/transferred");
+
+        _depositProposals[destinationChainID][depositNonce] = DepositProposal({
+            _destinationChainID: destinationChainID,
+            _depositNonce: depositNonce,
+            _dataHash: dataHash,
+            _yesVotes: new address[](0),
```

Comment:
May be better to initialise this as `_yesVotes: new address[](1)`
then instead of `_depositProposals[destinationChainID][depositNonce]._yesVotes.push(msg.sender);` to append the vote, just assign it like `_depositProposals[destinationChainID][depositNonce]._yesVotes[0] = msg.sender`